geronimo-openapi-impl/src/main/java/org/apache/geronimo/microprofile/openapi/impl/processor/AnnotationProcessor.java

```diff
@@ -59,6 +59,7 @@
 import javax.ws.rs.core.Context;
 import javax.ws.rs.core.Response;
 
+import org.apache.cxf.common.util.StringUtils;
```

Comment:
Nop, we dont depend on cxf in the impl except for a single spi impl ;). Just duplicate this one please

Reply:
:+1: will remove, thanks